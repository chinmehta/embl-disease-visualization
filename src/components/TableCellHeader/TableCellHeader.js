import React from "react";

function TableCellHeader({ text }) {
  return (
    <>
      <th className="font-bold py-4 px-2 leading-none border border-gray-400">
        {text}
      </th>
    </>
  );
}

export default TableCellHeader;
