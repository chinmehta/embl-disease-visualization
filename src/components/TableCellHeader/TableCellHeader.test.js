import React from "react";
import renderer from "react-test-renderer";

import TableCellHeader from "./TableCellHeader";

it("renders when no text  is passed as prop", () => {
  const element = renderer.create(<TableCellHeader />).toJSON();

  expect(element).toMatchSnapshot();
});

it("renders when header text is passed as prop", () => {
  const element = renderer.create(<TableCellHeader text="EMBL" />).toJSON();

  expect(element).toMatchSnapshot();
});
