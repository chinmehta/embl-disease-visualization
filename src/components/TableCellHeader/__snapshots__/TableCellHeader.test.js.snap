// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders when header text is passed as prop 1`] = `
<th
  className="font-bold py-4 px-2 leading-none border border-gray-400"
>
  EMBL
</th>
`;

exports[`renders when no text  is passed as prop 1`] = `
<th
  className="font-bold py-4 px-2 leading-none border border-gray-400"
/>
`;
